styles/styles.css

```diff
@@ -56,32 +43,100 @@
-  --overlay-background-color: #eee;
-  --highlight-background-color: #ccc;
-  --volvo-text-gray: #53565A;
-  --volvo-text-black: #000;
-  --volvo-text-light-gray: #A7A8A9;
-  --text-color: var(--volvo-text-gray);
-  --btn-background-hover: #e1dfdd;
-  --btn-border-hover: #030333;
-  --input-background-color: #f1f0ef;
-
   /* fonts */
   --ff-volvo-novum-medium: 'VolvoNovum-Medium', 'VolvoNovum-Medium Fallback', sans-serif;
   --ff-volvo-novum: 'VolvoNovum', 'VolvoNovum Fallback', sans-serif;
   --ff-volvo-broadprodigital: 'VolvoBroadProDigital', 'VolvoBroadProDigital Fallback', sans-serif;
   --ff-fontawesome: 'fontawesome', 'fontawesome Fallback';
-  --body-font-family: var(--ff-volvo-novum);
-  --heading-font-family: var(--ff-volvo-novum-medium);
-  --fixed-font-family: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+  --font-family-body: var(--ff-volvo-novum);
+  --font-family-heading: var(--ff-volvo-novum-medium);
+  --font-family-fixed: 'Roboto Mono', menlo, consolas, 'Liberation Mono', monospace;
+
+  /* REDESIGN */
+
+  /* Volvo Broad Pro Digital, 60px, Uppercase, 3.6px,#53565A */
+  --f-large-title-font-size: 3.75rem;
+  --f-large-title-text-transform: uppercase;
+  --f-large-title-letter-spacing: 3.6px;
+
+  /* Volvo Novum Medium, 48px, -0.1px, #53565A */
+  --f-large-text-font-size: 3rem;
+  --f-large-text-letter-spacing: -0.1px;
+
+  /* Volvo Broad Pro Digital, 48px,Uppercase, 3.6px,#53565A */
+  --f-medium-title-font-size: 3rem;
+  --f-medium-title-text-transform: uppercase;
+  --f-medium-title-letter-spacing: 3.6px;
+
+  /* Volvo Novum Medium, 32px, , -0.1px, #53565A */
+  --f-medium-text-font-size: 2rem;
+  --f-medium-text-letter-spacing: -0.1px;
+
+  /* Volvo Novum Medium / Volvo Novum Regular, 24px, , -0.1px, #53565A */
+  --f-small-font-size: 1.5rem;
+  --f-small-letter-spacing: -0.1px;
+
+  /* Volvo Novum Medium, 16px, , -0.1px, #53565A */
+  --f-xsmall-font-size: 1rem;
+  --f-xsmall-letter-spacing: -0.1px;
+
+  /* Volvo Novum Medium / Volvo Novum Regular, 12px, , -0.1px, #53565A */
+  --f-xxsmall-font-size: 0.75rem;
+  --f-xxsmall-letter-spacing: -0.1px;
+
+  /* Volvo Novum Regular, 16px, -0.1px,  #53565A */
+  --f-body-text-font-size: var(--f-xsmall-font-size);
+  --f-body-text-letter-spacing: var(--f-xsmall-letter-spacing);
+
+  /* COLORS */
+  --c-black: #000;
+  --c-white: #fff;
+  --c-volvo-blue: #182871;
+  --c-dark-blue: #202A44;
+  --c-leaf-one: #C8E691;
+  --c-leaf-two: #A8D46B;
+  --c-leaf-three: #8FC54E;
+  --c-leaf-four: #78B833;
+  --c-teal-one: #B8DeD8;
+  --c-teal-two: #8DC9BF;
+  --c-teal-three: #66B3A6;
+  --c-teal-four: #50A294;
+  --c-flow-one: #396976;
+  --c-flow-two: #678C96;
+  --c-flow-three: #96B0B6;
+  --c-flow-four: #C3D2D6;
+  --c-cta-blue-default: #004FBC;
+  --c-cta-blue-hover: #0056D6;
+  --c-cta-blue-active: #0041A3;
+
+  /* Greyscale */
+  --c-grey-1: #E1DFDD;
+  --c-grey-2: #A7A8A9;
+  --c-grey-3: #888B8D;
+  --c-grey-4: #53565A;
+
+  /* 
+  --c-error-red: #C4001A;
```

Comment:
If not needed we can remove the commented code.

Reply:
they are in the styleguide, not sure if they will be used, green and red might be used in the forms, but yellow 🤔 ?

Reply:
I kept success & error colours, but removed the warning yellow.